compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/AMDAIEOps.cpp

```diff
@@ -469,6 +470,22 @@ ConnectionOp::getNpuCircularDmaCpyNdUser() {
   return npuDmaUsers[0];
 }
 
+AMDAIE::FlowOp ConnectionOp::getFlowOp() {
+  return dyn_cast_if_present<AMDAIE::FlowOp>(getFlow().getDefiningOp());
```

Comment:
Would returning `std::optinal<AMDAIE::FlowOp>` be better?

Reply:
Yeah, that would be more clear. Updated it.